.travis.yml

```diff
@@ -18,26 +18,46 @@ env:
 matrix:
   include:
     - os: linux
-      env: BROWSER=chrome BVER=stable
+      env: BROWSER=chrome BVER=stable TOPOLOGY=P2P
 
     - os: linux
-      env: BROWSER=chrome BVER=beta
+      env: BROWSER=chrome BVER=stable TOPOLOGY=SFU
 
     - os: linux
-      env: BROWSER=firefox BVER=stable
+      env: BROWSER=chrome BVER=beta TOPOLOGY=P2P
 
     - os: linux
-      env: BROWSER=firefox BVER=beta
+      env: BROWSER=chrome BVER=beta TOPOLOGY=SFU
 
-    - os: osx
-      sudo: required
-      osx_image: xcode8.3
-      env: BROWSER=safari BVER=unstable
+    - os: linux
+      env: BROWSER=firefox BVER=stable TOPOLOGY=P2P
+
+    - os: linux
+      env: BROWSER=firefox BVER=stable TOPOLOGY=SFU
+
+    - os: linux
+      env: BROWSER=firefox BVER=beta TOPOLOGY=P2P
+
+    - os: linux
+      env: BROWSER=firefox BVER=beta TOPOLOGY=SFU
+
+    # - os: osx
+    #   sudo: required
+    #   osx_image: xcode8.3
+    #   env: BROWSER=safari BVER=unstable TOPOLOGY=P2P
+
+    # - os: osx
+    #   sudo: required
+    #   osx_image: xcode8.3
+    #   env: BROWSER=safari BVER=unstable TOPOLOGY=SFU
 
   allow_failures:
-    - env: BROWSER=chrome BVER=stable
-    - env: BROWSER=chrome BVER=beta
-    - env: BROWSER=safari BVER=unstable
+    - env: BROWSER=firefox BVER=stable TOPOLOGY=P2P
```

Comment:
@markandrus Any reason we are allowing stable firefox build tests to fail?

Reply:
There are a few issues w/ Firefox today--we can tweak the allowed failures later.